test/integers.jl

```diff
@@ -88,4 +88,22 @@
         @test Polymake.Integer(1) << 2 == 4
         @test Polymake.Integer(4) >> 2 == 1
     end
+
+    @testset "weird BigInt issue" begin
+        test_f(n) = (k = Polymake.Integer(n); BigInt(k))
+        bn = test_f(5)
+        GC.gc();
+        @test bn == 5
```

Comment:
this guards against invalid pointer in BigInt:
without `deepcopy` in the definition of `BigInt(::Polymake.Integer)` `bn` shares the underlying data with `k` and after `GC.gc()` that data is gone, so `bn` contains invalid pointer and is no longer equal to what it was.

Reply:
i see